lib/request-base.js

```diff
@@ -3,6 +3,19 @@
  */
 
 /**
+ * Root reference for environment-specific globals.
+ */
+
+var root;
+if (typeof window !== 'undefined') { // Browser window
+  root = window;
+} else if (typeof self !== 'undefined') { // Web Worker
+  root = self;
+} else { // Other environments (node or other)
+  root = {FormData: require('form-data')};
```

Comment:
And actually, based on this it's probably OK to just do a direct `var FormData = require('form-data');` and let browserify do it's thing.